includes/Modules/Idea_Hub.php

```diff
@@ -730,4 +746,37 @@ private function is_idea_post( $post_id ) {
 		return is_array( $this->get_post_idea( $post_id ) );
 	}
 
+	/**
+	 * Adds .googlesitekit-idea-hub__draft class to idea posts on the posts page.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $classes An array of post class names.
+	 * @param array $class An array of additional class names added to the post.
+	 * @param int   $post_id The post ID.
+	 * @return array An array of post class names.
+	 */
+	private function update_post_classes( $classes, $class, $post_id ) {
+		global $post_type;
+
+		$current_screen = get_current_screen();
+		if ( is_null( $current_screen ) || 'edit-post' !== $current_screen->id || 'post' !== $post_type ) {
+			return $notices;
+		}
+
+		if ( $this->is_idea_post( $post_id ) ) {
+			$classes[] = 'googlesitekit-idea-hub__draft';
```

Comment:
Do we want to add a draft-specific class to all Idea Hub posts? Is this only relevant for drafts or is it for all posts created from IH ideas? Maybe it should be `post` instead or only applied if it is currently a draft?

Reply:
Updated to use the `googlesitekit-idea-hub__post` class.